### PR TITLE
Issue #1- User can create an entry

### PR DESCRIPTION
**Pull request by hannahhoanggg**

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -18,6 +18,12 @@
   width: 100%;
 }
 
+@media only screen and (min-width: 576px) {
+  .column-half {
+    flex-basis: 50%;
+  }
+}
+
```
Comment (shawnkost) [inline css/layout.css]: This shouldn't be needed. I think you added this because your zoom level was off

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+.heading {
+  background-color: #562b81;
+  color: white;
+  font-size: 8px;
+}
+
+.top-header {
+  font-size: medium;
```
Comment (shawnkost) [inline css/styles.css]: You don't want to use `medium`, because what is `medium` exactly? You should be using `rem` or `px`

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+label {
+  font-size: 15px;
+  font-weight: bold;
+}
+
+img {
+  width: 100%;
+  height: 100%;
```
Comment (shawnkost) [inline css/styles.css]: Remove this. You only need to set the `width`. The `height` will automatically adjust for us

File: index.html [hunk 4]
```diff
@@ -16,13 +16,76 @@
...
+              <div class="image-text">
+                <label for="image">Photo URL</label>
+              </div>
+              <div class="image-input">
+                <input
+                  required
+                  name="image"
+                  type="url"
+                  class="image"
+                  id="image-text" />
```
Comment (shawnkost) [inline index.html]: The `for` attribute of the `label` needs to match the `id` of the `input`.

File: index.html [hunk 5]
```diff
@@ -16,13 +16,76 @@
...
+              </div>
+            </div>
+            <div class="column-full save-button">
+              <button>SAVE</button>
+            </div>
+          </div>
+        </form>
+      </div>
     </main>
+    <div data-view="entries" class="hidden"></div>
```
Comment (shawnkost) [inline index.html]: This should still be inside of `main`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Hannah! Just a few tweaks requested here!

File: css/layout.css (hunk 1 above)
Comment (hannahhoanggg) [reply css/layout.css]: LOL makes sense now.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**